homeassistant/components/remote/__init__.py

```diff
@@ -118,6 +118,7 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
             vol.Optional(ATTR_DEVICE): cv.string,
             vol.Optional(ATTR_COMMAND): vol.All(cv.ensure_list, [cv.string]),
             vol.Optional(ATTR_COMMAND_TYPE): cv.string,
+            vol.Optional(ATTR_FREQUENCY): vol.Coerce(float),
```

Comment:
This PR is changing our entity model. Before a PR can be considered there needs to be approval in a discussion in our architecture repository.

https://github.com/home-assistant/architecture/discussions

https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model

Reply:
Possibly related existing discussion:

https://github.com/home-assistant/architecture/discussions/1076